### ISSUE TITLE
Chrome App Migration?

### ISSUE THREAD

**Issue by bgiesing** [cluracan/pocketfurc#25]
In case you haven't seen yet @cluracan, Google is discontinuing Chrome apps on every platform except Chrome OS. Any plans for this? Google says devs should convert it to either Progressive Web Apps or native packaging with Electron/NW.js.

https://developers.chrome.com/apps/migration

**Comment by merklejerk** [comment]
Yeah, this is unfortunate. Two years isn't a bad time frame, though I worry the app will lose visibility to new users as they downplay the chrome web store. The Android app may also be in jeopardy as the chrome-to-cordova build tools (CCA) also lose developers.  

A web app is only an option if Furcadia provides a web friendly endpoint, like via WebSockets or REST. Given their desire to transition to the web, this could actually happen.

So right now it's wait and see.
